packages/dicomImageLoader/src/imageLoader/internal/options.ts

```diff
@@ -17,11 +17,53 @@ let options: LoaderOptions = {
   imageCreated(/* image */) {
     // image created code
   },
+
+  getRetrieveOptions(transferSyntaxUid, retrieveTypeId = '') {
+    const { retrieveConfiguration } = this;
+    if (!retrieveConfiguration) {
+      return null;
+    }
+    transferSyntaxUid ||= 'unknown';
```

Comment:
transferSyntaxUid => transferSyntaxUID

Reply:
I believe you can write the whole thing as this 

```js
getRetrieveOptions(transferSyntaxUid = 'unknown', retrieveTypeId = '') {
  const { retrieveConfiguration } = this;
  if (!retrieveConfiguration) {
    return null;
  }

  const baseKey = retrieveTypeId ? `${transferSyntaxUid}-${retrieveTypeId}` : transferSyntaxUid;
  const defaultKey = retrieveTypeId ? `default-${retrieveTypeId}` : 'default';

  return retrieveConfiguration[baseKey] || retrieveConfiguration[defaultKey];
}
```

Reply:
It would be great to add documentation on the available options for retrieveTypeId whic based on the example below should be 'lossy' and 'final'?

Reply:
Done